apps/condo/domains/acquiring/schema/AcquiringIntegration.js

```diff
@@ -63,6 +79,17 @@ const AcquiringIntegration = new GQLListSchema('AcquiringIntegration', {
         delete: false,
         auth: true,
     },
+    hooks: {
+        validateInput: ({ resolvedData, context, addValidationError }) => {
+            if (!hasDvAndSenderFields(resolvedData, context, addValidationError)) return
+            const { dv } = resolvedData
+            if (dv === 1) {
+                // NOTE: version 1 specific translations. Don't optimize this logic
```

Comment:
please use explicit returns

Reply:
Linter says it's redundant and we decided not to increase warning amount :)